packages/migrate/commonsChunkPlugin/__snapshots__/commonsChunkPlugin.test.js.snap

```diff
@@ -0,0 +1,295 @@
+                    chunks: 'initial',
+                    enforce: true,
+                    minChunks: 2
+                },
+
+                vendor: {
+                    name: 'vendor',
+                    chunks: 'initial',
+                    enforce: true,
+                    minChunks: 2,
+                    test: 'vendor'
+                }
+            }
+        }
+    }
+}
+"
+`;
+
+exports[`commonsChunkPlugin transforms correctly using "commonsChunkPlugin-2" data 1`] = `
+"module.exports = {
+    entry: {
+		vendor: './src/vendors.js',
+	},
+
+    optimizations: {
+        splitChunks: {
+            cacheGroups: {
+                vendor: {
+                    name: 'vendor',
+                    chunks: 'initial',
+                    enforce: true,
+                    minChunks: 1,
+                    test: 'vendor'
+                }
+            }
+        }
+    }
+}
+"
+`;
+
+exports[`commonsChunkPlugin transforms correctly using "commonsChunkPlugin-3" data 1`] = `
+"module.exports = {
+    optimizations: {
+        splitChunks: {
+            cacheGroups: {
+                commons: {
+                    name: 'commons',
+                    chunks: 'initial',
+                    enforce: true,
+                    minChunks: 1,
+                    filename: 'commons.js'
+                }
+            }
+        }
+    }
+}
+"
+`;
+
+exports[`commonsChunkPlugin transforms correctly using "commonsChunkPlugin-4" data 1`] = `
+"module.exports = {
+    entry: {
+		main: './src/index.js',
+	},
+
+    optimizations: {
+        splitChunks: {
+            chunks: 'async',
+            minSize: 2000,
+
+            cacheGroups: {
+                main: {
+                    name: 'main',
+                    minChunks: 1,
+                    test: 'main'
+                }
+            }
+        }
+    }
+}
+"
+`;
+
+exports[`commonsChunkPlugin transforms correctly using "commonsChunkPlugin-5" data 1`] = `
+"module.exports = {
+    entry: {
+		main: './src/index.js',
+	},
+
+    optimizations: {
+        splitChunks: {
+            cacheGroups: {
+                main: {
+                    name: 'main',
+                    chunks: 'initial',
+                    enforce: true,
+                    minChunks: 1,
+                    test: 'main'
+                }
+            }
+        },
+
+        runtimeChunk: {
+            name: 'runtime'
+        }
+    }
+}
+"
+`;
+
+exports[`commonsChunkPlugin transforms correctly using "commonsChunkPlugin-6a" data 1`] = `
+"module.exports = {
+    entry: {
+		main: './src/index.js',
+	},
+
+    optimizations: {
+        splitChunks: {
+            cacheGroups: {
+                main: {
+                    name: 'main',
+                    chunks: 'initial',
+                    enforce: true,
+                    minChunks: 1,
+                    test: 'main',
+                    test: ({ resource }) => /node_modules/.test(resource)
```

Comment:
You can't have `test` multiple times in the config.

You could be merged this way:

``` js
test: module => {
  if(module.getChunks().some(chunk => chunk.name === 'main')) return true;
  const fn = ({ resource }) => /node_modules/.test(resource);
  return fn(module);
}
```